docs/pages/switch.js

```diff
@@ -57,6 +57,196 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
           />
         </MainSection.Subsection>
       </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use a label to give the Switch context when possible."
+            defaultCode={`
+function SwitchExample2() {
+  const [switched, setSwitched] = React.useState(true);
+
+  return (
+    <Flex gap={2} alignItems="center" width="100%" maxWidth={300}>
+      <Flex.Item flex="grow">
+        <Label htmlFor="makesecret">
+          <Text weight="bold">Make this board secret</Text>
+        </Label>
+      </Flex.Item>
+      <Switch
+        onChange={() => setSwitched(!switched)}
+        id="makesecret"
+        switched={switched}
+      />
+    </Flex>
+  );
+}
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Truncate label text, instead allow it to wrap to form another line. "
```

Comment:
grammar nit: a comma isn't the right separator here. Either a period and start a new sentence with "Instead", or use a semicolon and add a comma after "instead"

Reply:
Done!